tests/syntax/test_specifiers.py

```diff
@@ -525,7 +525,9 @@ def test_visible_from_point():
     )
     for i in range(20):
         scene = sampleScene(scenario, maxIterations=10)
-        assert scene.egoObject.position.distanceTo(Vector(300, 200)) <= 2
+        assert scene.egoObject.position.distanceTo(Vector(300, 200)) <= 2 + math.sqrt(
+            3 * 0.5**2
```

Comment:
This is an awful black-ism: if moving to `2 + radius` doesn't fix it, please add `pt = Vector(300, 200)` above the loop or something similar so we can keep this mundane assertion a 1-liner. Likewise for the other test below.